CONTRIBUTING.md

```diff
@@ -90,18 +95,17 @@ python3 -m pip install -U --no-cache-dir  \
         --config-settings="--global-option=-I$(brew --prefix graphviz)/include/" \
         --config-settings="--global-option=-L$(brew --prefix graphviz)/lib/" \
```

Comment:
Do we need to make any significant changes to this command using uv?

Reply:
That seems reasonable. We can always add more sections as needed if people have issues.

Reply:
@eliotwrobson Great. Updated in 9e9808d9cca6e4359c93584c0273127bf17c1a00.